### PR TITLE
Rewrite `Reporting bad behaviour`

### PR DESCRIPTION
**Pull request by TicClick**
- textual overview of everything
- no dialogue-like FAQ (everything is in regular style)
- the intro probably could be improved

also, any thoughts on renaming the article? I feel that something short but descriptive would work better, but at the same time we need to maintain the hierarchy of `Reporting ...` > `(Abuse, Cheating)`

### CODE REVIEW

**Comment by TicClick** [comment]
additionally, image layout is wacky, but once we have infoboxes with zoom-in, it will be fixed.

File: wiki/Reporting_Bad_Behaviour/en.md [hunk 1]
```diff
@@ -1,116 +1,99 @@
...
-![The chat box with a !report command](img/report-command.jpg "Example of using the !report command")
-
-### Using the user report button in-game
-
-Reporting the user by this way only needs a type of reason and the custom text, unlike the `!report` command.
-
-![The list of available user options](img/report-user-1.png "Click the Report User button")
-
-![The user report overlay](img/report-user-2.png "Remember to add a reason here!")
+osu! has [a team of moderators](/wiki/People/The_Team/Global_Moderation_Team) who adhere to keeping the game and its ecosystem clean and safe. If you notice rule-breaking behaviour or inappropriate content, report it using one of the methods described below. Alternatively, you may contact any of the moderators online directly if your report wasn't properly handled within a reasonable amount of time — do so only as the last resort.
```
Comment (Walavouchey) [inline wiki/Reporting_Bad_Behaviour/en.md:3]: one team, group or organisation is still singular

```suggestion
osu! has [a team of moderators](/wiki/People/The_Team/Global_Moderation_Team) that adheres to keeping the game and its ecosystem clean and safe. If you notice rule-breaking behaviour or inappropriate content, report it using one of the methods described below. Alternatively, you may contact any of the moderators online directly if your report wasn't properly handled within a reasonable amount of time — do so only as the last resort.
```

File: wiki/Reporting_Bad_Behaviour/en.md [hunk 2]
```diff
@@ -1,116 +1,99 @@
...
 
-### Using the report button on the website
+Accidental reports are ignored. Users that purposedly send invalid reports are [silenced](/wiki/Silence).
```
Comment (Walavouchey) [inline wiki/Reporting_Bad_Behaviour/en.md:5]:
```suggestion
Accidental reports are ignored. Users who purposedly send invalid reports are [silenced](/wiki/Silence).
```

File: wiki/Reporting_Bad_Behaviour/en.md [hunk 3]
```diff
@@ -1,116 +1,99 @@
...
 
-Telling people to kill themselves is not a good thing to say.
+### Forum
 
-and many more...
+![](img/report-user-forum.png "The report button on a forum post")
 
-## Frequently Asked Questions
+1. Hover the cursor over the post.
+2. In a row of buttons that appear in the top right corner of the post, click the button with warning sign.
```
Comment (Walavouchey) [inline wiki/Reporting_Bad_Behaviour/en.md:66]:
```suggestion
2. In a row of buttons that appear in the top right corner of the post, click the button with a warning sign.
```

File: wiki/Reporting_Bad_Behaviour/en.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/Reporting_Bad_Behaviour/en.md]: hmm, just realised that it's ambiguous whether it's the "team" or the "moderators" doing the adhering, depending on where you put the emphasis while reading, so keep it or change it as you prefer.

File: wiki/Reporting_Bad_Behaviour/en.md [hunk 4]
```diff
@@ -1,116 +1,106 @@
...
-
-Reports are not case-sensitive so it is okay to use lowercase letters like `this_user_name`.
-
-![The chat box with a !report command](img/report-command.jpg "Example of using the !report command")
-
-### Using the user report button in-game
+osu! has [a team of moderators](/wiki/People/The_Team/Global_Moderation_Team) that adheres to keeping the game and its ecosystem clean and safe. If you notice rule-breaking behaviour or inappropriate content, report it using one of the methods described below. Alternatively, you may contact any of the moderators online directly if your report wasn't properly handled within a reasonable amount of time—do so only as the last resort.
 
-Reporting the user by this way only needs a type of reason and the custom text, unlike the `!report` command.
+Accidental reports are ignored. Users who purposedly send invalid reports are [silenced](/wiki/Silence).
```
Comment (Ephemeralis) [inline wiki/Reporting_Bad_Behaviour/en.md:5]:
```suggestion
Accidental reports are ignored. Users who deliberately send invalid reports can risk being [silenced](/wiki/Silence).
```

File: wiki/Reporting_Bad_Behaviour/en.md [hunk 5]
```diff
@@ -1,116 +1,106 @@
...
 
-![The list of available user options](img/report-user-1.png "Click the Report User button")
+## Special cases
 
-![The user report overlay](img/report-user-2.png "Remember to add a reason here!")
+| Nature of infringement | Point of contact |
+| :-- | :-- |
+| Mildly offensive private messages | **None**: [ignore the user](/wiki/Chat_Console#commands-list) instead. For permanent effect, use the in-game ignore list located in [options](/wiki/Options). |
```
Comment (Ephemeralis) [inline wiki/Reporting_Bad_Behaviour/en.md:11]:
```suggestion
| Offensive private messages | **None**: [ignore the user](/wiki/Chat_Console#commands-list) instead. For permanent effect, use the in-game ignore list located in [options](/wiki/Options). |
```

File: wiki/Reporting_Bad_Behaviour/en.md [hunk 6]
```diff
@@ -1,116 +1,106 @@
...
+## In-game chat
 
-Remember to add details to your report at the comments.
+### The !report command
 
-![](img/report-user-profile.png "The report button on a user's profile page")
+![](img/report-command.jpg "Example of using the !report command")
 
-![](img/report-user-forum.png "The report button at a user's post on the forum")
+Chat misbehaviour is the most common infringement. To report it, use the [`!report`](https://osu.ppy.sh/community/forums/topics/34843) command, preferably in the channel where infringement took place. Don't feel stressed: if your message starts with `!report` and a space, **it is only visible to moderators** and no one else in the chat.
```
Comment (Ephemeralis) [inline wiki/Reporting_Bad_Behaviour/en.md:24]:
```suggestion
Chat misbehaviour is the most common infringement. To report it, use the [`!report`](https://osu.ppy.sh/community/forums/topics/34843) command, preferably in the channel where infringement took place. Don't feel stressed: if your message starts with `!report` and a space, **it is only visible to moderators** and no one else in the chat. You can also use the report function via the in-game overlay as depicted below if you are still concerned.
```

**Review by Ephemeralis — CHANGES_REQUESTED** [review]
a few minor changes

**Review by Ephemeralis — APPROVED** [review]
looks good to me

**Review by TPGPL — APPROVED** [review]

**Pull request merged**